README.md

```diff
@@ -1,30 +1,28 @@
 # TornadoVM
 
-<img align="right" width="250" height="250" src="etc/tornadoVM_Logo.jpg">
+<img align="left" width="250" height="250" src="etc/tornadoVM_Logo.jpg">
 
-TornadoVM is a plug-in to OpenJDK and GraalVM that allows programmers to automatically run Java programs on
-heterogeneous hardware. TornadoVM currently targets OpenCL-compatible devices and it runs on multi-core CPUs, dedicated
-GPUs (NVIDIA, AMD), integrated GPUs (Intel HD Graphics and ARM Mali), and FPGAs (Intel and Xilinx).
+TornadoVM is a plug-in to OpenJDK and GraalVM that allows programmers to automatically run Java programs on heterogeneous hardware. 
+TornadoVM currently targets OpenCL, PTX and SPIR-V compatible devices and it runs on multi-core CPUs, dedicated
+GPUs (Intel, NVIDIA, AMD), integrated GPUs (Intel HD Graphics and ARM Mali), and FPGAs (Intel and Xilinx).
 
 
-TornadoVM currently has three backends: OpenCL, NVIDIA CUDA PTX and SPIR-V.
+TornadoVM currently has three backends that generate OpenCL C, NVIDIA CUDA PTX, and SPIR-V binary.
 Developers can chose which backends to install and run.
 
 ----------------------
 
 **Website**: [tornadovm.org](https://www.tornadovm.org)
 
-For a quick introduction please read the following [FAQ](tornado-assembly/src/docs/15_FAQ.md).
+For a quick introduction please read the following [FAQ](https://tornadovm.readthedocs.io/en/feat-documentation/faq.html).
```

Comment:
Will the link work once, we merge the `feat-documentation` branch?

Reply:
yes, it will work. When we do the release, we will point these to the master branch

Reply:
No, we need to change this after we merge.